crates/tako/src/messages/gateway.rs

```diff
@@ -57,10 +58,16 @@ pub enum FromGatewayMessage {
     CancelTasks(CancelTasks),
     GetTaskInfo(TaskInfoRequest),
     ServerInfo,
+    GetMonitoringEvents(EventRequest),
     GetOverview(OverviewRequest),
     StopWorker(StopWorkerRequest),
 }
 
+#[derive(Deserialize, Serialize, Debug)]
+pub struct EventRequest {
+    pub after_id: Option<u32>,
```

Comment:
Let's put a comment here that says how to use this attribute. And rename the struct to e.g. `MonitoringEventRequest` or just inline it, this name seems to be too generic.